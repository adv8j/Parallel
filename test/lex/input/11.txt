@Supervisor{
    call = call t2;
    call.join();
}
