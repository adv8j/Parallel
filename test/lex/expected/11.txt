SUPERVISOR
LBRACE
CALL
ASSIGN
CALL
IDENTIFIER
SEMICOLON
CALL
DOT
JOIN
LPAREN
RPAREN
SEMICOLON
RBRACE
